src/main/java/xyz/nucleoid/plasmid/widget/SidebarWidget.java

```diff
@@ -126,6 +132,25 @@ public Content writeLine(String line) {
             return this.writeRawLine(line);
         }
 
+        public Content writeFormattedTranslated(Formatting formatting, String key, Object... args) {
+            // This is probably a really awful way of doing this, but simply calling .formatted() on a TranslatableText
```

Comment:
Hmm- the solution here might be to change `getLineForPlayer` to apply the text style